Checkpoint-Profile-Lookup.md

```diff
@@ -74,9 +92,12 @@ return "No such contact";
 - `"likes"` is found within the first object, so the second `if` statement returns true
 - The value of `"likes"` is returned - `"Pizza", "Coding", "Brownie Points"`
 
-
+####Second solution
+(Declarative programming)
+Pretty much the same done in the previous solution, except without telling the machine how to do its job.
```

Comment:
Pretty dismissive.  If it's basically the same thing, why do we even care?  Why is this solution better?
>  except without telling the machine how to do its job.

This is not meaningful commentary.   Explain what is going on and why we'd do this or don't add the solution at all.